include/evaluators/blas2_tree_evaluator.hpp

```diff
@@ -109,8 +109,7 @@ struct Evaluator<PrdRowMatVctMultExpr<LHS, RHS1, RHS2, RHS3>, Device> {
         r1(Evaluator<RHS1, Device>(expr.r1)),
         r2(Evaluator<RHS2, Device>(expr.r2)),
         r3(Evaluator<RHS3, Device>(expr.r3)),
-        nThr(2)
-  {}
+        nThr(2) {}
```

Comment:
why 2? a comment is needed explaining why 2, possibly need to ask the original author